normits_demand/utils/general.py

```diff
@@ -2157,25 +2150,24 @@ def is_in_string(vals: Iterable[str],
     return False
 
 
-def get_compiled_matrix_name(matrix_format: str,
-                             user_class: str,
-                             year: str,
-                             trip_origin: str = None,
-                             mode: str = None,
-                             ca: int = None,
-                             tp: str = None,
-                             csv: bool = False,
-                             compress: bool = False,
-                             suffix: str = None,
-                             ) -> str:
+def get_compiled_matrix_name(
```

Comment:
How is this function different to `get_dist_name`? 

I realise you might not be the best person to answer this question - but an insight into what you've discovered could be useful here.
I imagine they're similar functions used by different parts of the code. Is there any obvious reason why?

Reply:
It seems likely thst whoever wrote this (I think Matt) possibly forgot they'd already made such a similar function, and at a minimum this function should inherit from get_dist_name. Possibly out of scope for the current PR to tidy up.